### DISCUSSION TITLE
Add a category to search artists, albums and songs all at once

### DISCUSSION THREAD

**Discussion by repomansez** [navidrome/navidrome#3555] [Ideas]
I would like to propose the addition of a "Search" category to Navidrome, similar to the functionality found in most streaming platforms. The idea is to allow users to search for artists, albums, and songs at once, with the results displayed in that order: Artists first, followed by Albums, and then Songs.

While I would be interested in contributing to the implementation of this feature, I currently lack experience with Go. From a user perspective, it seems to be the only major missing feature that I find somewhat limiting.

Thank you for considering this suggestion.

**Comment by deluan** [comment]
This is planned in the new UI, see #255

**Reply by repomansez** [reply]
> This is planned in the new UI, see #255

That's great! Thanks :)

We both hate people who ask this, but I'm going to ask: is there an ETA on when that'll be ready?
